src/BrandEmbassyCodingStandard/ruleset.xml

```diff
@@ -235,9 +236,6 @@
     <!-- Requires use of null coalesce operator when possible -->
     <rule ref="SlevomatCodingStandard.ControlStructures.RequireNullCoalesceOperator"/>
 
-    <!-- Forbid dead code -->
-    <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements"/>
```

Comment:
Why do we not want this anymore?

Reply:
The rule was removed by the author, who now recommends using phpstan, which has become more adept at addressing this kind of issue since the original code sniffer rule was introduced. You can read about it more here: [PHP Stan Blog - Detecting Unused Private Properties, Methods, and Constants](https://phpstan.org/blog/detecting-unused-private-properties-methods-constants )